src/runtime_src/core/common/query_requests.h

```diff
@@ -66,9 +66,14 @@ enum class key_type
   rom_time_since_epoch,
 
   xclbin_uuid,
+  group_topology,
+  memstat,
```

Comment:
What is the different between `memstat` and `memstat_raw`?

Reply:
one is human readable form and other is raw data